__tests__/app.test.js

```diff
@@ -150,6 +153,14 @@ describe('Test GET api/article/:article_id/comments endpoint', () => {
           expect(article['body']).toBeString()
           expect(article['created_at']).toBeString()
         })
+        expect(body[0]).toEqual({
+          comment_id: 5,
+          body: 'I hate streaming noses',
+          article_id: 1,
+          author: 'icellusedkars',
+          votes: 0,
+          created_at: '2020-11-03T21:00:00.000Z'
+        })
```

Comment:
You should also check that the comments are sorted. I can see you are checking that the first comment in the array is what you expect which is good logic. However, it would be more dynamic and robust to use something like the toBeSortedBy matcher (jest-sorted)